package.json

```diff
@@ -33,19 +33,25 @@
     "pug": "^2.0.3"
   },
   "devDependencies": {
+    "babel-eslint": "~8.2.3",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.3",
     "babel-preset-env": "^1.7.0",
     "babel-preset-flow": "^6.23.0",
     "cross-env": "^5.1.5",
-    "eslint": "^4.19.1",
-    "eslint-config-standard": "^11.0.0",
+    "eslint": "~4.19.1",
+    "eslint-config-standard": "~11.0.0",
+    "eslint-config-standard-react": "~6.0.0",
     "eslint-plugin-flowtype": "^2.46.3",
-    "eslint-plugin-import": "^2.12.0",
+    "eslint-plugin-jest": "~21.15.0",
+    "eslint-plugin-prefer-object-spread": "~1.2.1",
+    "eslint-plugin-react": "~7.7.0",
+    "eslint-plugin-standard": "~3.0.1",
```

Comment:
when do you use `~` and when `^` ?

Reply:
I switched back to the node default ^ in the webapp. No need to be consistent. I makes no diference when you execute: `yarn upgrade --latest`.

http://fredkschott.com/post/2014/02/npm-no-longer-defaults-to-tildes/